semantic_conventions/metrics/process-runtime-jvm-metrics.yaml

```diff
@@ -122,6 +122,13 @@ groups:
     instrument: updowncounter
     unit: "{class}"
 
+  - id: metric.process.runtime.jvm.cpu.time
+    type: metric
+    metric_name: process.runtime.jvm.cpu.time
+    brief: "CPU time used by the process."
+    instrument: counter
+    unit: "s"
```

Comment:
Looking at the links below https://docs.oracle.com/en/java/javase/17/docs/api/jdk.management/com/sun/management/OperatingSystemMXBean.html#getProcessCpuTime() says

>  The returned value is of nanoseconds precision but not necessarily nanoseconds accuracy

I wonder if we should use `ns` here instead?

Reply:
Otel spec has mostly standardized on decimal seconds for time representations.